save-orchestrator/src/main/resources/application-kubernetes.properties

```diff
@@ -0,0 +1,12 @@
+orchestrator.backend-url=http://backend/internal
+orchestrator.kubernetes.apiServerUrl=http://kubernetes.default.svc
+orchestrator.kubernetes.serviceAccount=${POD_SERVICE_ACCOUNT}
+orchestrator.kubernetes.current-namespace=${POD_NAMESPACE}
+orchestrator.kubernetes.agent-namespace=save-agent
+server.shutdown=graceful
+management.endpoints.web.exposure.include=*
+management.server.port=5101
+orchestrator.agent-settings.heartbeat-url=http://orchestrator/heartbeat
+orchestrator.agent-settings.debug=true
+logging.level.com.saveourtool.save.orchestrator.kubernetes=DEBUG
+logging.level.org.springframework=DEBUG
```

Comment:
I think that all properties from Spring can be moved to application.properties

Reply:
it's related to k8s. better to have them here.

Reply:
done. All releted spring props moved to main properties file